plugins/item_tasks/web_client/templates/paginateTasksWidget.pug

```diff
@@ -1,10 +1,10 @@
 .g-task-list-header
   .g-task-pagination
 
-.list-group.g-task-list-container
+.g-list-group.g-task-list-container
```

Comment:
I'll consider this to be a bugfix of a typo that should have been caught in the initial review of #2192 .